docker-compose-separate-containers.yml

```diff
@@ -1,23 +1,25 @@
-version: '2'
+version: '3'
```

Comment:
Thought version 3 is list as the recommended version
https://docs.docker.com/compose/compose-file/compose-versioning/